src/baseline_bots/parsing_utils.py

```diff
@@ -148,6 +148,7 @@ def daidefy_suborder(dipnet_suborder: str) -> str:
                     f"({' '.join(convoy_map[dipnet_order_tokens[0] + dipnet_order_tokens[1]])})"
                 )
             else:
+                daide_order.append(())
```

Comment:
Could you instead write `continue` here? This would skip adding this daide order itself in case the route does not exist.

Reply:
Sure